content/id/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,416 @@
+
+Untuk informasi lebih lanjut tentang mekanisme keamanan pada Linux, silahkan lihat
+[ikhtisar fitur keamanan pada Kernel Linux](https://www.linux.com/learn/overview-linux-kernel-security-features)
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Mengatur konteks keamanan untuk Pod
+
+Untuk menentukan aturan keamanan pada Pod, masukkan bagian `securityContext`
+dalam spesifikasi Pod. Bagian `securityContext` adalah sebuah objek
+[PodSecurityContext](/docs/reference/generate/kubernetes-api/{{<param"version">}}/#podsecuritycontext-v1-core).
+Aturan keamanan yang kamu tetapkan untuk Pod akan berlaku untuk semua Container dalam Pod tersebut.
+Berikut sebuah berkas konfigurasi untuk Pod yang memiliki volume `securityContext` dan `emptyDir`:
+
+{{< codenew file="pods/security/security-context.yaml" >}}
+
+Dalam berkas konfigurasi ini, bagian `runAsUser` menentukan bahwa dalam setiap Container pada
+Pod, semua proses dijalankan oleh ID pengguna 1000. Bagian `runAsGroup` menentukan grup utama dengan ID 3000 untuk
+semua proses dalam setiap Container pada Pod. Jika bagian ini diabaikan, maka ID grup utama dari Container
+akan berubah menjadi _root_(0). Berkas apa pun yang dibuat juga akan dimiliki oleh pengguna dengan ID 1000 dan grup dengan ID 3000 ketika `runAsGroup` ditentukan.
+Karena `fsGroup` ditentukan, semua proses milik Container juga merupakan bagian dari grup tambahan dengan ID 2000.
+Pemilik volume `/data/demo` dan berkas apa pun yang dibuat dalam volume tersebut adalah grup dengan ID 2000.
+
+Buatlah Pod tersebut:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/security/security-context.yaml
+```
+
+Periksa apakah Container dari Pod sedang berjalan:
+
+```shell
+kubectl get pod security-context-demo
+```
+Masuk ke _shell_ dari Container yang sedang berjalan tersebut:
+
+```shell
+kubectl exec -it security-context-demo -- sh
+```
+
+Pada _shell_ kamu, lihat daftar proses yang berjalan:
+
+```shell
+ps
+```
+
+Keluarannya menunjukkan bahwa proses dijalankan oleh pengguna dengan ID 1000, yang merupakan nilai dari bagian `runAsUser`:
+
+```shell
+PID   USER     TIME  COMMAND
+    1 1000      0:00 sleep 1h
+    6 1000      0:00 sh
+...
+```
+
+Pada _shell_ kamu, pindah ke direktori `/data`, dan lihat isinya:
+
+```shell
+cd /data
+ls -l
+```
+
+Keluarannya menunjukkan bahwa direktori `/data/demo` memiliki grup dengan ID 2000, yang merupakan
+nilai dari bagian `fsGroup`.
+
+```shell
+drwxrwsrwx 2 root 2000 4096 Jun  6 20:08 demo
+```
+
+Pada _shell_ kamu, pindah ke direktori `/data/demo`, dan buatlah sebuah berkas didalamnya:
+
+```shell
+cd demo
+echo hello > testfile
+```
+
+Lihatlah daftar berkas dalam direktori `/data/demo`:
+
+```shell
+ls -l
+```
+
+Tampilan menunjukkan bahwa `testfile` memiliki grup dengan ID 2000, dimana merupakan nilai dari bagian `fsGroup`.
```

Comment:
```suggestion
Keluarannya menunjukkan bahwa `testfile` memiliki grup dengan ID 2000, dimana merupakan nilai dari bagian `fsGroup`.
```

Reply:
I think some people will translate "tampilan" as "view" in their mind. Let's use the more specific translation for "output"!

Reply:
Ok!